### DISCUSSION TITLE
Metric definitions for DSP

### DISCUSSION THREAD

**Discussion by DanielaZamoraUribe** [amzn/ads-advanced-tools-docs#22] [Q&A]
Hi there!

Is there any documentation or link where I can find the definition of the DSP metrics that are included in the DSP API reports? I am looking for something like this, which was nicely done for all the Sponsored Ads metrics: https://advertising.amazon.com/API/docs/en-us/reporting/v3/metrics. 

If it's not available, what would be the best way to get support in case one metric is not clear or we don't understand how it is calculated?. 

Thanks in advance!

**Comment by elizafarley** [comment]
Hi @DanielaZamoraUribe Unfortunately, we don't currently have any similar documentation for DSP reporting metrics. 

For now, it would be best to [open a JIRA ticket](https://amzn-clicks.atlassian.net/servicedesk/customer/portal/2/group/2) to request clarification on specific metrics.

**Comment by elizafarley** [comment]
I wanted to post an update here that we have added [metric descriptions](https://advertising.amazon.com/API/docs/en-us/reporting/dsp/metrics) for DSP reporting! Hopefully this helps you better understand the data, but please let us know if you have any feedback.
